config/config.go

```diff
@@ -211,6 +211,7 @@ type Config struct {
 	FqdnOnly            FqdnOnlyConfig            `yaml:"fqdnOnly"`
 	Filtering           FilteringConfig           `yaml:"filtering"`
 	Ede                 EdeConfig                 `yaml:"ede"`
+	Ecs                 EcsConfig                 `yaml:"ecs"`
```

Comment:
Rename to `ECS`  since it's an abbreviation?

Reply:
Could be done but wouldn't that also apply to `Ede` & `Fqdn`?  
I think renaming all of them to unifying the name pattern would be out of scope and maybe part of another PR.(?)

Reply:
Yeah agreed just commented since this is new code. And we have SUDN that follows the convention already.  
I'll make an issue and we can do it separately.

Reply:
Ok I'll rename it to `ECS` and we'll rename the others in a separate PR. 👍

Reply:
We can do this in another too, doesn't matter to me :)